src/actions/localization-actions.js

```diff
@@ -50,12 +51,16 @@ export const getLanguageDataAction = ({ language }) => {
 
 export const getLanguageAction = () => {
     return async (dispatch) => {
-        const language = await getLanguage();
-
-        return dispatch({
-            payload: { language },
-            type: "GET_LANGUAGE"
-        });
+        try {
+            const language = await getLanguage();
+            dispatch({
+                payload: { language },
+                type: "GET_LANGUAGE"
+            });
+            dispatch(getKeyboardLayoutsAction({ language }));
```

Comment:
Please dont add this here. Find where getLanguageAction is called in src/apis/localization.js and do this bellow that.